runner_manager/models/runner_group.py

```diff
@@ -268,6 +268,24 @@ def healthcheck(
             self.delete_runner(runner)
             log.info(f"Runner {runner.name} deleted")
 
+    def reset(self, github: GitHub):
+        """Reset runner group."""
+        for runner in self.get_runners():
+            if runner.id is not None:
+                (
+                    github.rest.actions.get_self_hosted_runner_for_org(
+                        self.organization, runner.id
+                    ).parsed_data
+                )
```

Comment:
We recently made some change into this function and now `update_from_github` does this API call.

```suggestion
```

Reply:
Yeah and I don't really understand why those parenthesis around it.